test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala

```diff
@@ -242,6 +242,26 @@ class SubsidyControllerSpec
         FakeRequest(POST, routes.SubsidyController.postClaimDate.url).withFormUrlEncodedBody(data: _*)
       )
 
+      "return to claims page with Error Message" when {
+        "entered date is not valid" in {
+          val updatedDate = DateFormValues("21", "20", "2021")
+          inSequence {
+            mockAuthWithEnrolmentAndValidEmail()
+            mockRetrieveUndertaking(eori1)(undertaking.some.toFuture)
+            mockGet[SubsidyJourney](eori1)(Right(subsidyJourney.some))
+            mockTimeProviderToday(fixedDate)
+          }
+          val result = performAction("day" -> updatedDate.day, "month" -> updatedDate.month, "year" -> updatedDate.year)
+          val document = Jsoup.parse(contentAsString(result))
+
+          status(result) shouldBe BAD_REQUEST
+          val findElement = document.getElementById("claim-date-error").text()
+          findElement.replace("Error: ", "") shouldBe messageFromMessageKey("add-claim-date.error.incorrect-format")
```

Comment:
can we do `findElement shouldBe "Error: " + messageFromMessageKey("add-claim-date.error.incorrect-format")` instead, no need to replace "Error: " as it is part of the title when we have an error

Reply:
updated